src/workers/event.rs

```diff
@@ -208,6 +208,87 @@ where
         Ok(())
     }
 
+    #[instrument(level = "debug", skip(self, actor), fields(actor_id = %actor.public_key, host_id = %actor.host_id))]
+    async fn handle_actor_scaled(
+        &self,
+        lattice_id: &str,
+        actor: &ActorScaled,
+    ) -> anyhow::Result<()> {
+        trace!("Scaling actor in store");
+        debug!("Fetching current data for actor");
```

Comment:
~Nit: are both of these logs needed?~ Nvm, I see we do this everywhere, just confused why

Reply:
Yeah good point, the back to back makes it a little bit weirder but it is following the other pieces of code where we debug log around fetching data.

Reply:
I think that, once upon a time, there was stuff in between them. These ones could probably be combined, but I can see places where we might want to log something super deep at a trace level and then something else at a debug level